### ISSUE TITLE
Error loading "On Hold" orders

### ISSUE THREAD

**Issue by bporcelli** [bporcelli/simplesalestax#27]
A customer reports that they are getting the following fatal error every time they try to access an "On Hold" order in WP Admin:

```
Message Uncaught Error: Call to a member function getCertificateID()
on string in ABSPATH/wp-content/plugins/simplesalestax/includes/admin/class-sst-admin.php:111
Stack trace: #0
ABSPATH/wp-admin/includes/template.php(1304):
SST_Admin::output_tax_metabox(Object(WP_Post), Array) #1
ABSPATH/wp-admin/edit-form-advanced.php(635):
do_meta_boxes(Object(WP_Screen), 'side', Object(WP_Post)) #2
ABSPATH/wp-admin/post.php(197):
include('/home/zxcraftco...') #3 {main} thrown
File ABSPATH/wp-content/plugins/simplesalestax/includes/admin/class-sst-admin.php
```
**Pertinent Facts**
- The customer is running WooCommerce 3.6.5 and SST 6.0.6
- All affected orders were created in WP Admin
- All affected orders were created after Simple Sales Tax was installed

**Comment by bporcelli** [comment]
So far I'm unable to reproduce this. I've asked for temporary admin access so I can investigate further.

**Comment by bporcelli** [comment]
Upon further investigation I found that all affected orders had corrupted/invalid metadata. Specifically, _wootax_packages and _wootax_exempt_cert both had the value 's:1:"1";'.

Interestingly, a new On Hold order placed today did not seem to be affected. My theories right now are:

1. The affected orders existed when SST was installed. All update routines (starting from v1) were applied, thus corrupting the order metadata.
2. The problem surfaces when an On Hold order is created from the backend but the tax isn't calculated.
3. The problem is related to one of the other WooCommerce plugins installed on the reporting customer's site, perhaps the Auth.Net payment gateway or the Manual Payment plugin. I've downloaded all likely candidates for further testing.

I think (3) is most probable, but further investigation is needed.
